src/test/groovy/org/prebid/server/functional/tests/UserSyncSpec.groovy

```diff
@@ -60,4 +60,24 @@ class UserSyncSpec extends BaseSpec {
         IFRAME         || BLANK
         IFRAME         || PIXEL
     }
+
+    def "PBS should return empty uid in usersync url when uid macro not resolve"() {
```

Comment:
"PBS should return empty uid in usersync url when uid macro is not present in config"